R/linear_pool_quantile.R

```diff
@@ -15,27 +15,22 @@ linear_pool_quantile <- function(model_out_tbl, weights = NULL,
                                  n_samples = 1e4,
                                  ...) {
   quantile_levels <- unique(model_out_tbl$output_type_id)
+  agg_args <- c(list(x = quote(.data[["pred_qs"]]), probs = as.numeric(quantile_levels)))
 
   if (is.null(weights)) {
+    weight_by_cols <- NULL
     group_by_cols <- task_id_cols
-    agg_args <- c(list(x = quote(.data[["pred_qs"]]), probs = as.numeric(quantile_levels)))
   } else {
     weight_by_cols <- colnames(weights)[colnames(weights) != weights_col_name]
-
     model_out_tbl <- model_out_tbl |>
       dplyr::left_join(weights, by = weight_by_cols)
 
-    agg_args <- c(list(x = quote(.data[["pred_qs"]]),
-                       weights = quote(.data[[weights_col_name]]),
-                       normwt = TRUE,
-                       probs = as.numeric(quantile_levels)))
-
     group_by_cols <- c(task_id_cols, weights_col_name)
   }
 
   sample_q_lvls <- seq(from = 0, to = 1, length.out = n_samples + 2)[2:n_samples]
```

Comment:
Very minor point -- if the intent was to collect `n_samples` samples from each model, maybe we should change this line to grab entries at `[2:(n_samples + 1)]`:

```
  sample_q_lvls <- seq(from = 0, to = 1, length.out = n_samples + 2)[2:(n_samples + 1)]
```

Reply:
(fwiw, this feels like something that i likely introduced)

Reply:
I think you were the one to add this line, so I'll defer to you

Reply:
let's make the change, it's a minor point but nice to clean it up